bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/HtmlPrinter.java

```diff
@@ -116,8 +116,10 @@ public static String printSummary(final Altair altair) {
         return String.format(
             "r0(%.0fnm) = %.3f m\n" +
             "Strehl = %.3f\n"+
-            "FWHM of an AO-corrected core = %.3f arcsec\n",
-            altair.getWavelength(), altair.getr0(), altair.getStrehl(), altair.getAOCorrectedFWHM());
+            "FWHM of an AO-corrected core = %.3f arcsec\n"+
```

Comment:
This technically looks fine, but it would be cleaner if you moved some of the calculations outside of the `String.format(...)`.